site/assets/scss/_anchor.scss

```
@@ -1,11 +1,16 @@
-.anchorjs-link {
+.anchor-link {
+  padding-left: .175rem;
```

```suggestion
  padding: 0 .175rem;
```
Makes the focus indicator better balanced.